packages/components/src/dimension-control/stories/index.tsx

```diff
@@ -0,0 +1,44 @@
+/**
+ * External dependencies
+ */
+import type { ComponentMeta, ComponentStory } from '@storybook/react';
+/**
+ * Internal dependencies
+ */
+import { DimensionControl } from '..';
+import sizes from '../sizes';
+
+/**
+ * WordPress dependencies
+ */
+import { desktop, tablet, mobile } from '@wordpress/icons';
+
+export default {
+	component: DimensionControl,
+	title: 'Components (Experimental)/DimensionControl',
+	argTypes: {
+		onChange: { action: 'onChange' },
+		value: { control: { type: null } },
+		icon: {
+			control: { type: 'select' },
+			options: [ '-', 'desktop', 'tablet', 'mobile' ],
+			mapping: {
+				'-': undefined,
+				desktop,
+				tablet,
+				mobile,
+			},
+		},
+	},
```

Comment:
Without those settings, Storybook won't show the description and default value from the props' JSDocs

```suggestion
	},
	parameters: {
		controls: { expanded: true },
		docs: { source: { state: 'open' } },
	},
```

| Before | After |
|---|---|
| ![Screenshot 2023-03-02 at 13 52 10](https://user-images.githubusercontent.com/1083581/222433937-0cbdbf58-5be9-4157-9a6c-63a776793b00.jpg) | ![Screenshot 2023-03-02 at 13 53 13](https://user-images.githubusercontent.com/1083581/222434188-b23a251b-5b01-4ef1-9a13-489fcffa061b.jpg) |